qt/docker-compose-apalis-imx6.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.9"
+services:
+  app:
+    image: lucastor/cppqml-template:3.2.0
```

Comment:
@otaviocp who is responsible for these `lucastor/` namespace and where are the images coming from?

Reply:
This image was built on top of our [cppQML](https://github.com/toradex/vscode-torizon-templates/tree/dev/cppQML). We put it in a personal Docker Hub account just to have things working quickly. Ideally, we would have this image available and maintained on the Torizon repo. That way, we could leverage the Torizon team's pipelines to keep this image functional and up-to-date.

Reply:
@leonheldattoradex what we do here? This PR seems stalled, probably it isn't clear what and who should act.

Reply:
I'll push these images to the `torizon-extras` repo, just haven't got around to it.

Reply:
Can @otaviocp do it instead? He'll have to modify the Compose files then anyway

Reply:
Sure, if someone else has the bandwidth, that's ok with me.

Reply:
Sure. I can work on it. 

@leonheldattoradex How can I push images to the `torizonextras ` repo?